### ISSUE TITLE
Error running the example

### ISSUE THREAD

**Issue by jasonbosch** [jacobbien/trac#10]
Hello.
I think trac will be quite useful for something I want to do but I haven't been able to get it to run correctly. When I try to run the example, R crashes. I'm using R 4.1.1.

> > fit <- trac(ztr, ytr, A = sCD14$A, min_frac = 1e-2, nlam = 30)
> 
>  *** caught segfault ***
> address (nil), cause 'memory not mapped'
> 
> Traceback:
>  1: py_call_impl(callable, dots$args, dots$keywords)
>  2: prob$solve()
>  3: trac(ztr, ytr, A = sCD14$A, min_frac = 0.01, nlam = 30)

**Comment by jasonbosch** [comment]
I was hoping you had seen the error before. I've never used reticulate before and I only added that to use trac. It could be a problem there. If so, I'll have to find a way to test it. I am currently running Arch Linux with the 5.14.11-arch1-1 kernel.

**Comment by muellsen** [comment]
Hej again, I think we are sure now that the problem you describe is independent of trac but rather a Python/R interfacing problem. Could you maybe check whether you can call Python from your R  via reticulate for a simpler task, see, e.g., here https://github.com/compbiomed/singleCellTK/wiki/Calling-Python-with-Reticulate

**Comment by muellsen** [comment]
We will also add a short Tutorial to reticulate/c-lasso installation in the future.

**Comment by muellsen** [comment]
One thing you can try is to use the use_python command in reticulate prior to running trac:
https://rstudio.github.io/reticulate/reference/use_python.html
E.g., in my case for example, I have installed c-lasso in a specific conda environment and tell R prior to running trac
where it is!

library(reticulate)
# conda activate r-reticulate
use_python("~/.conda/envs/classo/bin/python",required = TRUE)

**Comment by jasonbosch** [comment]
Thank you for all the advice. It is an R/Python problem. I am able to run a c-lasso example correctly using the Python terminal. If I run the exact same thing through reticulate, it crashes. I'm not sure what I can do about that, but it's not an issue with trac. Sorry for the disruption.

**Comment by shu251** [comment]
Wanted to chime in here! I was running into a similar issue where R and python were not communicating. This also impacted use of c-lasso. I run R/RStudio via conda. Sharing steps I took to troubleshoot this (I'm running R v4.1.0):

First installed r-reticulate in environment I am using "r_4.1"
```
conda activate r_4.1
conda install -c conda-forge r-reticulate
pip install c-lasso
```
Opened R in this environment to help R and python talk to each other in this environment.
```
library(reticulate)
use_python("~/anaconda3/envs/r_4.1/bin/python", required = TRUE)
```

To install trac itself, needed to add some dependencies manually.
```
# install.packages("remotes")
# install.packages("devtools")

devtools::install_version("hsm", version = "0.2.0")
remotes::install_github("jacobbien/ggb")
devtools::install_github("jacobbien/trac")
```

**Comment by jasonbosch** [comment]
I forgot about this report and should have closed it sooner. I tried trac again this year and eventually solved the problem, which was entirely my fault. It was due to installing numpy with pip. Arch Linux is a bit particular about it's packages and there is an Arch-specific version of numpy in the official repositories. Once I reinstalled numpy from the Arch repository, everything functioned correctly.
